withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -2,7 +2,7 @@
 **  PowerOn Name:       BANNO.CHECK.WITHDRAW.V1.POW
 **  Letterfile Name:    BANNO.CHECK.WITHDRAW.V1.CFG
 **
-**  Copyright 2020-2022 Jack Henry and Associates
+**  Copyright 2020-2022 Jack Henry
 **
 **  Modification History:
 **   Ver. 1.0.0  03/13/2020: T. Kainz - Original Programming
```

Comment:
At the end of the Print section of the code in Line 328 and 329 Here:
IF BNODEBUGMODE=BNODEBUGMODEOFF THEN
  NEWLINE
I’m not sure this new line is needed as the other conditions have a NEWLINE, but if you have tested,
and having an extra NEWLINE does not lead to an issue, then it is fine
 PRINT "  }"
END [PRINT]

At Line 640 and 641:
ELSE IF PARAMNAME="MIN" THEN
  PARAMMINWDLIMIT=MONEY(VALUE(PARAMVALUE))
I don’t think this would happen, but should there be validation here to make it max at 999,999.99?

At Line 645 and 646:
ELSE IF PARAMNAME="MAX" THEN
        PARAMMAXWDLIMIT=MONEY(VALUE(PARAMVALUE))
should there be validation here to make it max at 999,999.99?

I can’t tell if withdraw-by-check-json-contract.md is still in the submission (those changes you made are important) as the last commit only has changes to the CFG and the PowerOn, so if commits only show programs that have changes, then its fine. 

The Code Looks Good. The list I provided are clean up suggestions.